src/rpad/pybullet_envs/pm_suction.py

```diff
@@ -263,6 +265,201 @@ def run_trial(
     )
 
 
-class PMSuctionGymEnv(gym.Env):
-    # TODO(eric): Implement the following functions.
-    pass
+# max simulation steps for control
+TIMEOUT = 1000
+
+
+class PMSuctionDemoEnv:
+    def __init__(self, obj_id, pm_dataset_path, gui):
+        self.obj_id_str = obj_id
+        self.obj = PMObject(pm_dataset_path / obj_id)
+        self.renderer = PybulletRenderer()
```

Comment:
this you shouldn't use! caution. basically because it creates its own internal pybullet simulation, ignoring yours. Only use PMRenderEnv